views/index.ejs

```diff
@@ -48,6 +48,9 @@
         <button id="play-pause" type="button" class="btn btn-success">
           <span class="glyphicon glyphicon-play" aria-hidden="true"></span>
         </button>
+        <button id="record" type="button" class="btn btn-success">
```

Comment:
As explained in #27, we want to export the loop created, not record the stream.
So I propose to put the button somewhere else, maybe at the top horizontal bar.

Reply:
I'm personally in favor of the record button next to the play button. It allows to export more than one loop, and the bpm is know so export to a DAW is easy.